Source/Core/Core/HW/WiimoteEmu/Speaker.cpp

```diff
@@ -106,18 +106,17 @@ void Wiimote::SpeakerData(wm_speaker_data* sd)
 		return;
 	}
 
-	// Speaker Pan
-	unsigned int vol = (unsigned int)(m_options->settings[4]->GetValue() * 100);
-
+	// Speaker Pan using Constant Power Pan Law
+	double pan = m_options->settings[4]->GetValue() / 1.27;
+	double pan_prime = 3.14159265358979323846 * (pan + 1) / 4.0;
 	unsigned int sample_rate = sample_rate_dividend / m_reg_speaker.sample_rate;
 	float speaker_volume_ratio = (float)m_reg_speaker.volume / volume_divisor;
-	unsigned int left_volume = (unsigned int)((128 + vol) * speaker_volume_ratio);
-	unsigned int right_volume = (unsigned int)((128 - vol) * speaker_volume_ratio);
 
-	if (left_volume > 255)
-		left_volume = 255;
-	if (right_volume > 255)
-		right_volume = 255;
+	unsigned int left_volume = (unsigned int)(256 * cos(pan_prime) * speaker_volume_ratio);
+	unsigned int right_volume = (unsigned int)(256 * sin(pan_prime) * speaker_volume_ratio);
+
+	left_volume = MathUtil::Clamp(left_volume, 0u, 256u);
```

Comment:
actually 256 breaks mixer.cpp line 284 to 285, but I think that code should be changed to volume/256 anyways, since mixer.h asserts volume is 0-256